docs/9.0/index.md

```diff
@@ -15,9 +15,9 @@ layout: default
 
 ## Usage
 
-### Parsing a document
+### Parsing a CSV document
 
-Accessing some records from a given CSV documents.
+Access 25 records from a CSV document saved on the filesystem.
```

Comment:
the only remark because everything else is 👌🏾 I am not sure if we should mention the number of record. Apart from that all is OK

Reply:
I will merge the PR and update this